exampleSite/content/docs/shortcodes/index.md

```diff
@@ -90,6 +90,26 @@ Call to action
 Call to action
 {{< /button >}}
 
+## Carousel
+
+`carousel` is used to showcase multiple images in an interactive and visually appealing way. This allows a user to slide through multiple images while only taking up the vertical space of a single one. All images are displayed using the full width of the parent component and using a ratio of `16:9`.
+
+<!-- prettier-ignore-start -->
+| Parameter | Description                                               |
+| --------- | --------------------------------------------------------- |
+| `images`  | This shortcode receives how many image paths as you want. |
```

Comment:
I am not so sure about this table as it kinda suggests that there is a named parameter called `images`. I think the examples should suffice.

Reply:
Saw your changes thanks